code/game/objects/items/weapons/legcuffs.dm

```diff
@@ -127,9 +120,39 @@
 					SSblackbox.record_feedback("tally", "handcuffs", 1, type)
 
 			else
-				L.apply_damage(trap_damage, BRUTE)
+				if(istype(L, /mob/living/simple_animal/hostile/bear))
+					L.apply_damage(trap_damage * 2.5, BRUTE)
+				else
+					L.apply_damage(trap_damage * 1.75, BRUTE)
 	..()
 
+/obj/item/restraints/legcuffs/beartrap/on_found(mob/finder)
+	if(!armed)
+		return FALSE
+	spring_trap(finder, IED, sig)
```

Comment:
why is IED a parameter when its a var stored on the beartrap. same with signaller.